packages/trader/src/index.html

```diff
@@ -1,6 +1,21 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <script>
+        performance.mark('start');
```

Comment:
This will have to move to `core/src/index.html` once PR #531 is merged.

Reply:
@mahdipourziaei-binary I added performance observer and mark to core's index.html